src/lib/lightning/clightning/clightningService.ts

```diff
@@ -122,6 +124,7 @@ class CLightningService implements LightningService {
       id: toPubKey,
       satoshis: amount,
       feeRate: '253perkw', // min relay fee for bitcoind
+      announce: !isPrivate,
```

Comment:
`c-lightning` uses the `announce` field to mark a channel as private. However, when you retrieve channels from `c-lightning` the channel is not marked as private (`private` field is always `false`).

Reply:
Do you know if the root cause of this problem is with the `c-lightning-REST` API or with `lightningd`? We should open an issue in their repos if the RPC isn't working as expected.

I don't think it's a good idea to display the "Make private" checkbox and then show a non-private channel after it's created. That makes it seem like Polar is broken.

Reply:
I don't know where the issue lies. I will have to look deeper into it. I believe I am using the right field here yet the channel is not private no matter what value I send it.

> I don't think it's a good idea to display the "Make private" checkbox and then show a non-private channel after it's created. That makes it seem like Polar is broken.
Yeah, I agree. I just wanted to create a PR anyway to start the discussion and figure out what's wrong in the process.

Reply:
Ok cool, let me know if you'd like me to help troubleshoot in any way.